### PR TITLE
Use same logo in night mode and private mode

### PR DESCRIPTION
**Pull request by NotWoods**
Actual optimization - night mode uses the same logo asset as private now. 

The about page has been updated so that the geckoview version is on a separate line. A-C is now denoted by an emoji.

Nothing else to see here...

### CODE REVIEW

File: app/src/main/res/layout/fragment_about.xml [hunk 1]
```diff
@@ -3,27 +3,27 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent">
+            xmlns:app="http://schemas.android.com/apk/res-auto"
+            xmlns:tools="http://schemas.android.com/tools"
+            android:layout_width="fill_parent"
```
Comment (colintheshots) [inline app/src/main/res/layout/fragment_about.xml]: Nit: fill_parent is deprecated by match_parent. I know you only changed the indent, but we should avoid ancient, deprecated attributes.

File: app/src/main/res/layout/fragment_about.xml [hunk 2]
```diff
@@ -3,27 +3,27 @@
...
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:app="http://schemas.android.com/apk/res-auto" xmlns:tools="http://schemas.android.com/tools"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        tools:context="org.mozilla.fenix.settings.AboutFragment">
+        android:layout_height="wrap_content">
 
         <ImageView
             android:id="@+id/wordmark"
-            android:importantForAccessibility="no"
```
Comment (colintheshots) [inline app/src/main/res/layout/fragment_about.xml]: Should we read a content description for our logo?

Comment (NotWoods) [reply app/src/main/res/layout/fragment_about.xml]: "About Firefox Preview" is written below the logo, so I thought that a content description isn't needed here.

File: app/src/main/res/layout/fragment_about.xml (hunk 1 above)
Comment (pocmo) [reply app/src/main/res/layout/fragment_about.xml]: I'm surprised `lint` doesn't point that out.

**Pull request merged**